python/nav/mibs/juniper_mib.py

```diff
@@ -171,10 +171,12 @@ def get_memory_usage(self):
             [OPERATING_DESCR, OPERATING_MEM, OPERATING_BUF]
         )
         for row in slots.values():
-            total = row[OPERATING_MEM]
+            # Convert total from MB to B
+            total = row[OPERATING_MEM] * (1024**2)
             if total:
-                # Due to operating buffer being in percent multiply by 1000 to get bytes
-                used = row[OPERATING_BUF] * total * 10000
+                # Operating buffer is in percentage
+                # Convert from MB to B
+                used = (row[OPERATING_BUF] / 100) * total * (1024**2)
```

Comment:
Still don't agree completely on the math here. `total` is now already a number of bytes, thanks to line 175, and yet you multiply it by `1024**2` *again* ...

Also, @hmpf mentions a good idea over my should here: This would be even more readable if you added a constant at the top of the file, `MEGABYTE = 1024**2` and multiplied by this constant. You might not even need the code comments with this.